### DISCUSSION TITLE
Potential problem serializing events...

### DISCUSSION THREAD

**Discussion by telkins** [spatie/laravel-webhook-server#77] [Q&A]
I'm not sure that this is an actual problem with the package, so please keep that in mind as you read. 🤓

I have a listener for `WebhookCallSucceededEvent` that I use to log successful webhook calls.  It works fine...until I define it as "queueable" by having the listener implement `ShouldQueue`.  Once I do that, I can no longer access the response body.

My guess is that access to it is lost because under the hood it appears to be streaming a resource.  I'm guessing that this is "lost" when serializing the listener for the queue.

Assuming I'm not missing something and this is an actual problem, I'm still not sure if this is laravel-webhook-server's problem.  One could argue that the underlying `Response` functionality should deal with it better, or I should find a way to make it work, or whatever.

Anyway, I'd like to put this out there and see what people have to say about it.  😜

For now, I've just "de-queued" the listener.  It's only logging the event....and the webhook already takes place within a job queue. 😅

Thanks...! 🚀

**Comment by tm1000** [comment]
@telkins Thanks! I was trying to figure out this same issue myself.

**Comment by gustavo-ticto** [comment]
I'm facing the same issue here. But in my case, i can't "de-queue" the listener that sends the webhook

**Comment by isaac-souza** [comment]
@telkins your comments just saved my day, many thanks!
